.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,44 @@
-**Description:**
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
 
+## Description
+<!--- Describe your changes. -->
 
-**Related issue (if applicable):** fixes #
+## Types of Changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
```

Comment:
I think that this is weird. This means it's either one of the above, but also is a breaking change. Instead this should be just a single line entry just like type.

**Type of change:** bug fix / feature
**Breaking change:** yes / no